### PR TITLE
J hardaway fix fonts

### PR DESCRIPTION
**Pull request by jeromehardaway**
I fixed all the serif fonts to be sans-serif per the style guide

### CODE REVIEW

File: src/assets/css/main.css [hunk 1]
```diff
@@ -362,7 +362,7 @@ p.featured-content {
 }
 blockquote {
   border: 0;
-  font-family: 'Gotham';
+  font-family: 'Gotham' sans-serif;
```
Comment (Cameron-Porter) [inline src/assets/css/main.css:365]: Does this work without the comma separating the values?

File: src/assets/css/custom.css [hunk 2]
```diff
@@ -176,7 +176,7 @@
 
 #cause-title {
   font-size: 1.5rem;
-  font-family: 'Gotham';
+  font-family: 'Gotham' sans-serif;
```
Comment (Cameron-Porter) [inline src/assets/css/custom.css:179]: Same question as below

**Review by Cameron-Porter — APPROVED** [review]

File: src/assets/css/main.css (hunk 1 above)
Comment (axecopfire) [reply src/assets/css/main.css]: Does need to be comma separated. I checked in the Chrome debug tool.
https://developer.mozilla.org/en-US/docs/Web/CSS/font-family

**Comment by axecopfire** [comment]
There are 16 other instances of `font-family: 'Gotham';` in the `main.css` file. Is there a reason we're not updating those as well? Also, I think this could be a good opportunity to remove so many instances of overriding font-family
image

File: src/assets/css/custom.css (hunk 2 above)
Comment (jeromehardaway) [reply src/assets/css/custom.css]: Fixed it.

File: src/assets/css/main.css [hunk 3]
```diff
@@ -1814,7 +1814,7 @@ uneditable-input:focus {
   outline: 0 none;
 }
 .story-title {
-  font-family: 'Gotham';
+  font-family: 'Gotham'sans-serif;
```
Comment (axecopfire) [inline src/assets/css/main.css:1817]: missing comma

**Review by axecopfire — APPROVED** [review]

**Comment by jeromehardaway** [comment]
> 

Just made those changes. Thanks for pointing that out.

File: src/assets/css/main.css [hunk 4]
```diff
@@ -335,7 +335,7 @@ h1.small-top-pad {
   text-transform: uppercase;
 }
 .featured-heading h2 {
-  font-family: 'Gotham';
+  font-family: 'Gotham', sans-serif, sans-serif;
```
Comment (axecopfire) [inline src/assets/css/main.css:338]: Too many `sans-serif`

File: src/assets/css/main.css [hunk 5]
```diff
@@ -174,10 +174,10 @@ h4,
 h5,
 h6 {
   color: var(--element-3);
-  font-family: 'Gotham', sans-serif;
+  font-family: 'Gotham', sans-serif, sans-serif;
 }
 .font-alt {
-  font-family: 'Gotham', sans-serif;
+  font-family: 'Gotham', sans-serif, sans-serif;
```
Comment (axecopfire) [inline src/assets/css/main.css:180]: Too many `sans-serif`

File: src/assets/css/main.css [hunk 6]
```diff
@@ -1814,7 +1814,7 @@ uneditable-input:focus {
   outline: 0 none;
 }
 .story-title {
-  font-family: 'Gotham';
+  font-family: 'Gotham', sans-serifsans-serif;
```
Comment (axecopfire) [inline src/assets/css/main.css:1817]: This one's broken

**Pull request merged**